dockstore-webservice/src/main/java/io/dockstore/webservice/languages/LanguageHandlerInterface.java

```diff
@@ -609,11 +651,284 @@ static DockerSpecifier determineImageSpecifier(String image, DockerImageReferenc
                     Set<Image> dockerHubImages = getImagesFromDockerHub(repo, specifierName, imageSpecifier);
                     dockerImages.addAll(dockerHubImages);
                 }
+            } else if (registryFound == Registry.GITHUB_CONTAINER_REGISTRY) {
+                splitDocker = image.split("/"); // ghcr.io/<repo>/<image>:1 -> ["ghcr.io", "repo-name", "image-name:1"]
+                int minNumOfNameComponents = 3;
+                if (splitDocker.length >= minNumOfNameComponents) {
+                    String owner = splitDocker[1];
```

Comment:
Nit-pick, do you want to call this repo instead? I think you call it repo on line 655, in the example URI, and it got confusing looking back and forth trying to correlate this variable to your example URI.